### DISCUSSION TITLE
Overriding hydra_runner config from command line

### DISCUSSION THREAD

**Discussion by oytunturk** [NVIDIA-NeMo/NeMo#3764] [Q&A] [answered]
Is there a way to provide ”config_path” and “config_name” parameters from command line for the @hydra_runner call in model trainer examples? These parameters are currently hardcoded in model trainer python code such as tacotron2.py, hifigan.py, etc which make it a bit impractical when training many models using different config files. I’m looking for a solution like:
`python tacotron2.py ... config_path=x config_name=y`

Thanks in advance!

**Comment by titu1994** [comment, marked as answer]
--config-path and --config-name are two two command line args.

**Reply by titu1994** [reply]
I'm pretty sure it's noted right in the first primer notebook, plus the documentation itself.

**Reply by oytunturk** [reply]
Thanks!
